products/views.py

```diff
@@ -1,3 +1,63 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.http      import JsonResponse
+from django.views     import View
+from products.models  import Product
+from users.utils      import login_decorator
+
+class ProductDetailView(View):
+    @login_decorator
+    def post(self, request, id):
+        try:
+            data               = json.loads(request.body)
+            product_id         = id
+            product            = Product.objects.get(id = product_id)
+            product_images     = product.productimages.filter(product = product)
+            information_images = product.informationimages.filter(product = product)
```

Comment:
```suggestion
            product_images     = product.productimages.all()
            information_images = product.informationimages.all()
```
product를 통해서 이미지를 찾는 부분은 잘 했습니다.
하나만 고치자면, 이미 특정 지은 product로부터 이미지를 찾기 시작했기 때문에 filter가 걸렸다고 생각하시면 좋을 것 같습니다.
제가 리뷰 드린 코드를 한 번 shell에서 테스트 해보세요.

Reply:
넵 확인해본 후 해당 코드처럼 수정했습니다.